modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,372 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-ocm-role.adoc
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-sts-aws-requirements-attaching-boundary-policy_{context}"]
+= Permission boundaries for the Installer role
+
+You can apply a policy as a _permissions boundary_ on an installer role.
+You can use an AWS-managed policy or a customer-managed policy to set the boundary for an Amazon Web Services(AWS) Identity and Access Management (IAM) entity (user or role). The combination of policy and boundary policy limits the maximum permissions for the user or role. ROSA includes a set of three prepared permission boundary policy files, with which you can restrict permissions for the installer role since changing the installer policy itself is not supported.
+
+
+The permission boundary policy files are as follows:
+
+* The _core_ boundary policy file contains the minimum permissions needed for ROSA (classic architecture) installer to install a cluster.
+If you need to install a cluster with the most restricted permissions, only use this permission boundary policy and there would be no VPC or PrivateLink (PL) permissions for the installer role.
+* The _VPC_ boundary policy file contains the minimum permissions needed for ROSA (classic architecture) installer to create/manage the VPC. It does not include permissions for PL or core installation. If you need to install a cluster with enough permissions for the installer to install the cluster and create/manage the VPC, but you do not need to set up PL, then use the core and VPC boundary files together with the installer role.
+* The _PrivateLink (PL)_ boundary policy file contains the minimum permissions needed for ROSA (classic architecture) installer to create the AWS PL with a cluster. It does not include permissions for VPC or core installation. If you want to install a cluster with PL, it requires a custom VPC, so you would need the PL and the core installation permissions.
+
+When using the permission boundary policy files, the following combinations apply:
+
+* No permission boundary policies = the current installer policy permissions.
+* *Core only* = most restricted; VPC and PL permissions are not available to the installer role.
+** Installer cannot create or manage the VPC or PL.
+** You must have a customer-provided VPC, and PL is not available, which means this cannot be for a private cluster.
```

Comment:
```suggestion
** You must have a customer-provided VPC, and PrivateLink is not available. 
```

Reply:
@fahlmant I wanted to call out the "no private cluster" concept here so it's super clear.    Do you think it's enough to have it as suggested?

Reply:
PrivateLink and Private clusters are two different things for ROSA Classic and OSD.

Reply:
implemented Taylor suggestion of removing ref to private cluster.